shared/util/feature-flags.desktop.tsx

```diff
@@ -33,6 +34,7 @@ const inAdmin: {[K in keyof FeatureFlags]?: boolean} = {
   chatIndexProfilingEnabled: true,
   connectThrashCheck: true,
   dbCleanEnabled: true,
+  tabletSupport: true,
```

Comment:
this doesn't make sense, this is for desktop

Reply:
this is meant to mean whether tablet support is public, not whether the current device is a tablet